tests/test_other.py

```diff
@@ -439,6 +439,28 @@ def test_combining_object_files(self):
     self.run_process([EMCC, 'combined.o', '-o', 'combined.o.js'])
     self.assertContained('side got: hello from main, over', self.run_js('combined.o.js'))
 
+  def test_combining_object_files_into_archive(self):
```

Comment:
Should this be "_from_archive"?    Are the object being merged ones that live inside that archive?

(other than that I think we are ready to land this)

Reply:
Yes, it is. Sorry about that...